addons/hr_holidays/tests/test_global_leaves.py

```diff
@@ -48,53 +48,54 @@ def setUpClass(cls):
 
         cls.global_leave = cls.env['resource.calendar.leaves'].create({
             'name': 'Global Leave',
-            'date_from': date(2022, 3, 7),
-            'date_to': date(2022, 3, 7),
+            'date_from': "2022-03-07 00:00:00",
+            'date_to': "2022-03-07 23:59:59",
```

Comment:
Were the tests failing before that? If not, I feel like that diff can be avoided